.gitignore

```diff
@@ -1,11 +1,7 @@
 
 *.deb
 
-.theos/fakeroot
-
-.theos/last_package
-
-.theos/packages/*
+.theos
```

Comment:
This should be `.theos/*`, (`.theos/` would work too, we just want to specify that it is a directory)